edx-olive/configuration-old
# SE-1228: Set a smaller root volume size when building AMIs

As in the title - use a smaller root volume size for building AMIs. This has the aim to reduce the size of volumes launching in the autoscaling groups.

Attempting to configure it from here since currently the launch configurations do not override the disk size from the AMI, and this seemed like an easier route.

**Testing instructions**:

1. Build an AMI using this branch as the configuration version (I used stage, edxapp for the ami)
2. create a new cluster using this AMI
3. ssh into the instance created in the cluster
4. check the disks available (eg. `df -h` or `lsblk`) and confirm that the root device size is 20GB

(Alternately, for steps 3 and 4, you can find the new instance in the aws console, and navigate to the root block device volume from there.) 

**Reviewers**:
